### PR TITLE
fix(onboarding): add data-test id for integration testing

### PR DESCRIPTION
**Pull request by TMcMeans**
The type of this PR is: fix

### Description

This PR is part of the work that fixes the integration tests failing in integrity. Adding a `data-test-id` helps select the skip button in cypress tests. 

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by laurabeth — APPROVED** [review]
🔥

File: src/Components/Onboarding/Views/OnboardingWelcome.tsx [hunk 1]
```diff
@@ -61,6 +61,7 @@ export const OnboardingWelcome = () => {
               as={RouterLink}
               to="/"
               onClick={onClose}
+              data-test="onboarding-skip-button"
```
Comment (damassi) [inline src/Components/Onboarding/Views/OnboardingWelcome.tsx:64]: (totally no blocking trivial comment): worth noting that in a JS world there's no need to use hyphens in variable values and they're pretty foreign syntax-wise (and mostly a legacy html idiom). `data-test="onboardingSkipButton"` works just as well, matches rest of the code

Comment (TMcMeans) [reply src/Components/Onboarding/Views/OnboardingWelcome.tsx:64]: hey, fixing the integration tests now so ill update on a follow-up pr addressing new integration tests for onboarding flow.

Comment (damassi) [reply src/Components/Onboarding/Views/OnboardingWelcome.tsx:64]: no need to follow up! just a side comment for the future

**Pull request merged**